tests/e2e/api/account.test.ts

```diff
@@ -79,6 +78,84 @@ describe("account api", () => {
       expect(data).toHaveProperty("type");
       expect(data.type).toBe("0x1::account::Account");
     });
+
+    test("it fetches account transactions", async () => {
+      const config = new AptosConfig({ network: Network.LOCAL });
+      const aptos = new Aptos(config);
+      const senderAccount = Account.generate({ scheme: SigningScheme.Ed25519 });
+      await aptos.fundAccount({ accountAddress: senderAccount.accountAddress.toString(), amount: 1000000000 });
+      const bob = Account.generate({ scheme: SigningScheme.Ed25519 });
+      const rawTxn = await aptos.generateTransaction({
+        sender: senderAccount.accountAddress.toString(),
+        data: {
+          function: "0x1::aptos_account::transfer",
+          type_arguments: [],
+          arguments: [bob.accountAddress, new U64(10)],
+        },
+      });
+      const authenticator = aptos.signTransaction({
+        signer: senderAccount,
+        transaction: rawTxn,
+      });
+      const response = await aptos.submitTransaction({
+        transaction: rawTxn,
+        senderAuthenticator: authenticator,
+      });
+      const txn = await aptos.waitForTransaction({ txnHash: response.hash });
+      const accountTransactions = await aptos.getAccountTransactions({
+        accountAddress: senderAccount.accountAddress.toString(),
+      });
+      expect(accountTransactions[0]).toStrictEqual(txn);
+    });
+
+    test("it fetches account transactions", async () => {
```

Comment:
```suggestion
    test("it fetches account transaction count", async () => {
```
probably needs a name change here